p2panda-rs/src/document/document.rs

```diff
@@ -5,32 +5,21 @@ use std::fmt::{Debug, Display};
 
 use crate::document::error::DocumentBuilderError;
 use crate::document::materialization::{build_graph, reduce};
-use crate::document::{DocumentId, DocumentView, DocumentViewId};
+use crate::document::{DocumentId, DocumentViewFields, DocumentViewId};
 use crate::identity::PublicKey;
 use crate::operation::traits::{AsOperation, WithOperationId, WithPublicKey};
-use crate::operation::{Operation, OperationId};
+use crate::operation::{Operation, OperationId, OperationValue};
 use crate::schema::SchemaId;
 use crate::Human;
 
+use super::DocumentView;
```

Comment:
Maybe we keep this in the `crate` level import? It's strange that it suddenly pops up down here next to the other imports.